<launch>
    <node name="davinci_grasp_info_writer" pkg="cwru_davinci_grasp" type="davinci_grasp_info_writer" required="true">
        <rosparam command="load" file="$(find cwru_davinci_grasp)/config/dvrk_psm_grasp_needle_data.yaml"/>
        <param name="packPath" value="$(find cwru_davinci_grasp)"/>
        <param name="planning_group_name" value="psm_one"/>
        <param name="ee_group_name" value="psm_one_gripper"/>
    </node>
</launch>
